app/src/main/res/layout/fragment_settings_language.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/md_theme_surface"
+    android:orientation="vertical"
+    tools:context=".ui.main.settings.language.SettingsLanguageFragment">
+
+    <RadioGroup
+        android:id="@+id/rgLanguages"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/md_theme_primary">
+
+        <RadioButton
+            android:id="@+id/rbEnglish"
+            style="@style/RadioButtonItem"
+            android:layout_width="match_parent"
+            android:layout_height="60dp"
+            android:text="@string/settings_language_english" />
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:layout_marginHorizontal="10dp"
+            android:alpha="0.5"
+            android:background="@color/md_theme_secondary" />
+
+        <RadioButton
+            android:id="@+id/rbSerbian"
+            style="@style/RadioButtonItem"
+            android:layout_width="match_parent"
+            android:layout_height="60dp"
+            android:text="@string/settings_language_srpski" />
+    </RadioGroup>
+
+    <View
```

Comment:
Consider using `com.google.android.material.divider.MaterialDivider` instead of generic `View`